src/DeferredResult/DeferredResult.spec.ts

```diff
@@ -277,4 +277,73 @@ describe("DeferredResult", () => {
             }
         )
     })
+
+    describe("map", () => {
+        it("returns a Resolved with projected inner value if Resolved and Ok", () => {
+            const incr = (n: number) => n + 1
```

Comment:
🎨❓ do we need the `incr` binding in the other tests below? I see how it aids test readability here, but in the test cases where it's not used, we could just inline it, right? (similar question about the `Deferred` tests above)

Reply:
https://github.com/fp-toolkit/fp-toolkit/pull/66/commits/56461bc3042a4c0f1f28ced77ad111edc3816758